app/templates/releases/lts.hbs

```diff
@@ -12,34 +12,37 @@
 <h2>How to Install</h2>
 
 <div class="mb-2">
-  <Releases::HowToInstall
-    @project={{@model}}
-  />
+  <Releases::HowToInstall @project={{@model}} />
```

Comment:
It seems like your text editor might have changed the format unintentionally. Can we revert lines 15-73, since they are not related to removing 3.16 LTS?

Reply:
Ah yes, I ran the linter within VS, I'll revert those changes.